Changelog.md

```diff
@@ -4,15 +4,38 @@
 
 ### Ignition Rendering 6.0.0 (20XX-XX-XX)
 
+1. Add bounding box sensor API stubs
+    * [Pull request #420](https://github.com/ignitionrobotics/ign-rendering/pull/420)
+
+1. Segmentation Camera
+    * [Pull request #329](https://github.com/ignitionrobotics/ign-rendering/pull/329)
+    * [Pull request #419](https://github.com/ignitionrobotics/ign-rendering/pull/419)
```

Comment:
include the PR that's just merged? https://github.com/ignitionrobotics/ign-rendering/pull/443